packages/contentful--create-contentful-app/src/app-actions/javascript/build-actions.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const { readdirSync, readFileSync, writeFileSync, existsSync, mkdirSync } = fs;
+
+const buildActions = () => {
+  const buildFolderExists = existsSync('build');
+
+  if (!buildFolderExists) {
+    mkdirSync('build');
+  }
+
+  const actionsFiles = readdirSync('actions', {
+    encoding: 'utf-8',
+  });
+
+  const actions = actionsFiles.map((fileName) => {
+    return readFileSync(`actions/${fileName}`, {
+      encoding: 'utf-8',
+    }).toString();
+  });
+
+  mkdirSync('build/actions');
```

Comment:
This should create an actions folder, regardless of whether `build` exists or not.
```suggestion
  mkdirSync('build/actions', { recursive: true });
```

So lines 5-9 could be deleted.